### ISSUE TITLE
Documentation issue - installation

### ISSUE THREAD

**Issue by gawth** [raphael/goa#125]
Hi,

I started from http://goa.design/getting-started.html 
I installed following instructions from : http://goa.design/goagen.html

Neither one mentions use/dependency on goimports so initially I hadn't installed it.

The code gen completes successfully but the resulting code doesn't build due to incorrect imports - took me a while to stumble on the requirement for goimports.

I suggest you make the code gen step fail if goimports isn't installed.  Can you also update the docs on the web page we reference to it.

Finally, the example on the getting started needs updating, app.Bottle isn't valid, needs changing to app.GoaExampleBottle.

Thanks,
Alan

**Comment by raphael** [comment]
Thanks for the report! Is it possible that you might have been using an older version of `goagen`? (well older than when [this PR](https://github.com/raphael/goa/pull/116) got merged anyway). `goagen` is supposed to check for `goimports` and fail if it doesn't find it.

I'll update the example to reflect the latest type names, thanks!

**Comment by gawth** [comment]
Wow that was quick - I see docs are updated.

I'll try again on a fresh env.

Plus, are you aware that goagen doesn't work on windows?  When I try to do a go get it fails.  This thread has a discussion on the problem : https://github.com/fvbock/endless/issues/12

Thanks,
Alan

**Comment by raphael** [comment]
Ah I didn't know... would you care to create a new issue? I'll close this one as the docs are updated, thanks!

**Comment by gawth** [comment]
I got to the root of the problem.  I already had goimports installed.  My issues arose out of how I ran through the guide.  When I originally ran goagen I did so from the parent folder of cellar which then created app, main.go, etc.  I then moved them to where I wanted them not realising that the import statements in the generated code were incorrect.

I'll open up a fresh issue for the windows problem.
Ta
